### DISCUSSION TITLE
深圳职业技术学院更名

### DISCUSSION THREAD

**Discussion by bsdayo** [CollegesChat/university-information#182] [University]
根据[深圳政府在线](http://www.sz.gov.cn/cn/xxgk/zfxxgj/zwdt/content/post_10654075.html)，及多方面报道，深圳职业技术学院已在今年（2023年）升本成功，更名为“深圳职业技术大学”。

今年新发出的录取通知书也改成了“大学”二字，详见[官方公众号文章](https://mp.weixin.qq.com/s/MsCVzWun1GMS10jdRwD6qA)

涉及到的相关页面，希望可以进行修改：

https://cn.colleges.chat/universities/shen-zhen-zhi-ye-ji-zhu-xue-yuan/
https://cn.colleges.chat/archived/universities/shen-zhen-zhi-ye-ji-zhu-xue-yuan/

另外自己作为一名准备入学的深职新生，感谢作者创建这样一个项目，帮助很大 :heart:

**Comment by bsdayo** [comment]
@Rachel030219

**Comment by Rachel030219** [comment]
已更新！预计将于半小时内上线完毕

**Comment by bsdayo** [comment]
感谢！
